addons/website/models/website_menu.py

```diff
@@ -47,6 +48,7 @@ def _set_field_is_mega_menu(self):
     is_mega_menu = fields.Boolean(compute=_compute_field_is_mega_menu, inverse=_set_field_is_mega_menu)
     mega_menu_content = fields.Html(translate=html_translate, sanitize=False, prefetch=True)
     mega_menu_classes = fields.Char()
+    mega_menu_layout_template = fields.Char(default='website.s_mega_menu_multi_menus')
```

Comment:
A new field should not be necessary, can't the info be saved in the DOM just like for masonry? (even an unique class on the template root maybe?)